client/templates/course/toolbar/edit-course.js

```diff
@@ -10,3 +10,26 @@ Template.editCourse.events({
     Session.set('editingCourseId', courseId);
   }
 });
+
+Template.editCourse.helpers({
+	'owner': function() {
```

Comment:
This template helper is not related to the pull request  'profile settings and features', so needs to be part of a separate pull request.